src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
+          <img src="touxiang.jpg" alt="picture" style="text-align:center">
```

Comment:
FYI, the CSS stylings of a tag can be inherited from its ancestor tags and the letter C in CSS represents Cascading for this. Thus, for your use case, the same `text-align: center` styles applied to the sibling tags (h1, h2 and img) can be lifted up to the parent `div#top`.